### PR TITLE
add support for ost-images CI

### PR DESCRIPTION
**Pull request by michalskrivanek**
this will build a custom set of ost-images and then run
basic-suite-master on them

### CODE REVIEW

**Comment by michalskrivanek** [comment]
jenkins changes already applied, not fully verified yet

File: .github/README.md [hunk 1]
```diff
@@ -11,6 +11,9 @@
   - initial step - paste first and second part together. You must provide IP= and RHEL= strings at the top, and in Build Environment use secret file OST_APP_PRIVATE_KEY_FILENAME
   - inject variables from "vars"
   - conditinal step(multiple) if build is still succeeding at this point
+    - for ost-images runs only
```
Comment (tinez) [inline .github/README.md:14]: The indentation here is wrong, if the two points below were supposed to be indented let's end `for ost-images runs only` with `:`

**Review by tinez — APPROVED** [review]

File: lagofy.sh [hunk 2]
```diff
@@ -60,6 +60,7 @@ ost_destroy() {
         ) 9>/tmp/ost.lock
     fi
     [[ -s "$PREFIX/sshd_pid" ]] && { echo "killing IPv6 sshd proxy"; kill $(cat "$PREFIX/sshd_pid"); }
+    [[ -d "$OST_REPO_ROOT/custom-ost-images" ]] && { echo "remove custom ost images"; rm -rf "$OST_REPO_ROOT/custom-ost-images"; }
```
Comment (tinez) [inline lagofy.sh:63]: I don't see the `--justdb` option used anywhere. Don't we need to uninstall those images when destroying?

File: common/helpers/ost-images.sh [hunk 3]
```diff
@@ -1,15 +1,30 @@
 #!/bin/bash -xe
 
-OST_IMAGES_DIR=${OST_IMAGES_DIR:=/usr/share/ost-images}
+if [[ -n "$CUSTOM_OST_IMAGES_REPO" ]]; then
+  echo "Installing custom ost-images from $CUSTOM_OST_IMAGES_REPO"
+  sudo dnf -y --installroot="$OST_REPO_ROOT/custom-ost-images" --releasever=/  install bash
```
Comment (tinez) [inline common/helpers/ost-images.sh:5]: Will it work even if the directory doesn't exist?

File: lagofy.sh (hunk 2 above)
Comment (michalskrivanek) [reply lagofy.sh:63]: no, --installroot is basically a regular full chroot, that's why we need to add bash first too

File: common/helpers/ost-images.sh (hunk 3 above)
Comment (michalskrivanek) [reply common/helpers/ost-images.sh:5]: yes

**Pull request merged**